extensions/spark/kyuubi-spark-authz/src/main/resources/table_command_spec.json

```diff
@@ -470,6 +470,27 @@
   } ],
   "opType" : "ALTERTABLE_REPLACECOLS",
   "queryDescs" : [ ]
+}, {
+  "classname" : "org.apache.spark.sql.catalyst.plans.logical.ReplaceData",
+  "tableDescs" : [ {
+    "fieldName" : "originalTable",
```

Comment:
`ReplaceData.table` is replaced by `RowLevelOperationTable` without properties, so `originalTable`is more suitable.